Makefile

```diff
@@ -37,27 +41,18 @@ endif
 BUILDMNT = /go/src/$(GOTARGET)
 BUILD_IMAGE ?= golang:1.11.2
 
-HYPERFED_TARGET = bin/hyperfed-linux
-CONTROLLER_TARGET = bin/controller-manager-linux
-KUBEFED2_TARGET = bin/kubefed2-linux
-
-CONTROLLER_TARGET_NATIVE = bin/controller-manager
+HYPERFED_TARGET = bin/hyperfed
+CONTROLLER_TARGET = bin/controller-manager
+KUBEFED2_TARGET = bin/kubefed2
 
 LDFLAG_OPTIONS = -ldflags "-X github.com/kubernetes-sigs/federation-v2/pkg/version.version=$(GIT_VERSION) \
                       -X github.com/kubernetes-sigs/federation-v2/pkg/version.gitCommit=$(GIT_HASH) \
                       -X github.com/kubernetes-sigs/federation-v2/pkg/version.gitTreeState=$(GIT_TREESTATE) \
                       -X github.com/kubernetes-sigs/federation-v2/pkg/version.buildDate=$(BUILDDATE)"
-BUILDCMD_HYPERFED = CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o $(HYPERFED_TARGET) $(VERBOSE_FLAG) $(LDFLAG_OPTIONS)
-BUILDCMD_CONTROLLER = CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o $(CONTROLLER_TARGET) $(VERBOSE_FLAG) $(LDFLAG_OPTIONS)
-BUILDCMD_KUBEFED2 = go build -o $(KUBEFED2_TARGET) $(VERBOSE_FLAG) $(LDFLAG_OPTIONS)
-
-BUILDCMD_CONTROLLER_NATIVE = CGO_ENABLED=0 go build -o $(CONTROLLER_TARGET_NATIVE) $(VERBOSE_FLAG) $(LDFLAG_OPTIONS)
+TARGET_ENV_linux = GOARCH=amd64 GOOS=linux
+TARGET_ENV_darwin = GOARCH=amd64 GOOS=darwin
```

Comment:
we could avoid these 2 variables, if we just use GOOS=$(HOST_OS)

Reply:
done